accord-core/src/main/java/accord/primitives/PartialTxn.java

```diff
@@ -55,6 +57,8 @@ default boolean covers(Participants<?> participants)
         return covering().containsAll(participants);
     }
 
+    boolean covers(PartialTxn txn);
```

Comment:
Coming up with a name for it was a struggle, because all these terms are overloaded, including 'superceding'. I scouted thesaurus to no satisfactory outcome.

Reply:
Renamed to `isEqualOrFuller()`, cheers.